content/language/R/develop.md

```diff
@@ -0,0 +1,229 @@
+---
+title: Use containers for R development
+keywords: R, local, development
+description: Learn how to develop your R application locally.
+---
+
+## Prerequisites
+
+Complete [Containerize a R application](containerize.md).
+
+## Overview
+
+In this section, you'll learn how to set up a development environment for your containerized application. This includes:
+
+- Adding a local database and persisting data
+- Configuring Compose to automatically update your running Compose services as you edit and save your code
+
+## Get the sample application
+
+You'll need to clone a new repository to get a sample application that includes logic to connect to the database.
+
+1. Change to a directory where you want to clone the repository and run the following command.
+
+   ```console
+   $ git clone https://github.com/mfranzon/r-docker-dev.git
+   ```
+
+2. In the cloned repository's directory, run `docker init` to create the necessary Docker files. Refer to the following example to answer the prompts from `docker init`.
+
+  ```console
+  $ docker init
+  Welcome to the Docker Init CLI!
+
+  This utility will walk you through creating the following files with sensible defaults for your project:
+    - .dockerignore
+    - Dockerfile
+    - compose.yaml
+    - README.Docker.md
+
+  Let's get started!
+
+  ? What application platform does your project use? Other
+
+  CREATED: .dockerignore
+  CREATED: Dockerfile
+  CREATED: compose.yaml
+  CREATED: README.Docker.md
+  ```
+
+## Add a local database and persist data
```

Comment:
This already exists in the compose.yaml in the repository

Reply:
Remove reference to file generated by `docker init`